tools/IoTEdgeDevOps/DevOpsLib/IoTEdgeReleaseDeployment.cs

```diff
@@ -16,7 +16,7 @@ public class IoTEdgeReleaseDeployment : IEquatable<IoTEdgeReleaseDeployment>
 
         public IoTEdgeReleaseDeployment(int id, int attempt, VstsDeploymentStatus status, DateTime lastModifiedOn, HashSet<IoTEdgePipelineTask> tasks)
         {
-            ValidationUtil.ThrowIfNonPositive(id, nameof(id));
+            ValidationUtil.ThrowIfNegative(id, nameof(id));
```

Comment:
Needed to change because we were getting a record with id 0. This seemed weird to me though. I logged the deployment that was trying to be created:
```
[{"id":0,"attempt":1,"deploymentStatus":0,"lastModifiedOn":"2020-03-09T18:37:54.4Z","releaseDeployPhases[0].deploymentJobs[0].tasks":null}]
```